recipes/django-socio-grpc/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "django-socio-grpc" %}
+{% set version = "0.19.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/django_socio_grpc-{{ version }}.tar.gz
+  sha256: 7b756ff1c7c304d4e7ea146e2a8b895bd0ad63f513e3db57c21ce29fbd6e9dd6
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

This should add the missing entry_point and fix that Windows failure. The odd thing is that I'm not even sure we need that entrypoint in the final package. It does seem like a build tool for running the tests. Also, that name is super generic and easy to clobber.

```suggestion
  number: 0
    entry_points:
   -  tests = "test_utils.load_tests:launch"
```